Documentation

AlexisDaniels opened the pull request:
This pull request concerns the addition of a documentation section + several other details to set up the package

AlexisDaniels committed:
Update README.md

AlexisDaniels committed:
Creating a package

AlexisDaniels committed:
added new file

AlexisDaniels committed:
Added documentation and a class

Now I can import the package and use the functions independently

AlexisDaniels commented on Crypto_bot_pkg/crypto_selection.py:
```diff
@@ -7,3 +8,22 @@
 def which_crypto():
     print("This function will allow you to choose between BTC, ETH, ADA or DOGE coins")
 
+=======
```
Here is where the assignment from this week begins - all other file updates were to allow the installation of the package

ToscaBeijaert replied:
You could add documentation to the which_crypto() after you've made the function of course.

ToscaBeijaert commented on Crypto_bot_pkg/crypto_selection.py:
```diff
@@ -0,0 +1,29 @@
+    print("This function will allow you to choose between BTC, ETH, ADA or DOGE coins")
+
+=======
+class crypto_bot:
```
classes should be with capital letters (Crypto_bot), sorry for the nitpicking

AlexisDaniels replied:
Don't be sorry, thanks for the style suggestion!

ToscaBeijaert commented on build/lib/Crypto_bot_pkg/crypto_selection.py:
```diff
@@ -0,0 +1,17 @@
+class crypto_bot:
```
again, make it Crypto_bot

ToscaBeijaert commented on build/lib/Crypto_bot_pkg/crypto_selection.py:
```diff
@@ -0,0 +1,17 @@
+
+    def which_crypto():
+        """
+        This is function 1
```
once you've made the function, don't forget to add parameters to the documentation

MarayahAy replied:
Yes I'd second that!

ToscaBeijaert reviewed (approved):
There was not that much to say. I guess just start writing the functions ;)

AlexisDaniels committed:
add your personal API address here

AlexisDaniels committed:
Installed SSL certificates - get inifinite messages

MarayahAy commented on build/lib/Crypto_bot_pkg/crypto_selection.py:
```diff
@@ -0,0 +1,17 @@
+        
+    def trade_bot():
+        """
+        This is function 2
```
I'd suggest that you briefly explain what the function does in the documentation as well.

AlexisDaniels committed:
Vignette for the crypto_bot package

This vignette was created using Jupyter notebooks. It contains information about how to use the GUI and some details about the RSI.
